tests/dash/app_styling.py

```diff
@@ -443,10 +443,10 @@ def layout():
                     "width": "100%"
                 },
                 style_data_conditional=[{
-                    "if": { "column_id": "Region", "filter": "Region eq str(Montreal)" },
+                    "if": { "column_id": "Region", "filter": "Region eq Montreal" },
```

Comment:
I thought we needed to bracket `{Region}` now?

Reply:
Updating.